base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,23 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(function, ArgumentTypes::Type, arguments...)
```

Comment:
> We usually don't capitalize argument names, perhaps change to argtypes? 

See the doc, for e.g., `ccall` I'd prefer someone open another PR to make it all uniform if wanted, but since `Types` follow the capitalization convention,  I think it makes sense to leave this as is. 

Also, ::Type should be ::Tuple?

No since, e.g., `Tuple{Int}` is a tuple type not a tuple